sputnikdao2/src/proposals.rs

```diff
@@ -27,6 +27,8 @@ pub enum ProposalStatus {
     Expired,
     /// If proposal was moved to Hub or somewhere else.
     Moved,
+    /// If proposal has failed when finalizing. Allowed to re-finalize again to either expire or approved.
+    Failed,
```

Comment:
Good idea keeping this more generic, as mine was `PayoutFailed`. Part of me likes it being a specific failure, but this is probably the better path.